cmd/provisioner-localpv/app/provisioner_hostpath.go

```diff
@@ -119,10 +119,10 @@ func (p *Provisioner) DeleteHostPath(pv *v1.PersistentVolume) (err error) {
 	glog.Infof("Deleting volume %v at %v:%v", pv.Name, node, path)
 	cleanupCmdsForPath := []string{"rm", "-rf"}
 	podOpts := &HelperPodOptions{
-		cmdsForPath: cleanupCmdsForPath,
-		name:        pv.Name,
-		path:        path,
-		nodeName:    node,
+		cmdsForPath:  cleanupCmdsForPath,
+		name:         pv.Name,
+		path:         path,
+		nodeHostname: node,
```

Comment:
should this be from GetNodeHostname()

Reply:
@kmova  This should be the `hostname` right?

Reply:
The `node` in this case, really contains the hostname. I have updated the code to change `node` to `hostname` and also renamed the method to make it explicit that hostname is being returned. 

Good catch!